packaging/elasticsearch_plugin.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+
+from ansible.module_utils.basic import *
+
+"""
+Ansible module to manage elasticsearch plugins
+(c) 2015, Mathew Davies <thepixeldeveloper@googlemail.com>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: elasticsearch_plugin
+short_description: Manage Elasticsearch plugins
+description:
+    - Manages Elasticsearch plugins.
+version_added: ""
+author: Mathew Davies (@ThePixelDeveloper)
+options:
+    name:
+        description:
+            - Name of the plugin to install
+        required: True
+    state:
+        description:
+            - Desired state of a plugin.
+        required: False
+        choices: [present, absent]
+        default: present
+    url:
+        description:
+            - Set exact URL to download the plugin from
+        required: False
+        default: None
+    timeout:
+        description:
+            - Timeout setting: 30s, 1m, 1h... (1m by default)
+        required: False
```

Comment:
default in description, but we want a field for use in formatting in ansible-doc and the docs.ansible.com site